storage/object/api-cli/bucket-policy.mdx

```diff
@@ -93,19 +101,19 @@ Bucket policies use a JSON-based access policy language and are composed of stri
 
 ## Statement
 
-**Description** 
+**Description**
 : `Statement` is an array that defines the rules that should be respected by our policy engine.
```

Comment:
Add that we can have several statements in a bucket policy?

Reply:
afaik you can include an array of statements